streaming/stream_manager.cc

```diff
@@ -164,11 +164,20 @@ void stream_manager::remove_progress(UUID plan_id) {
 }
 
 stream_bytes stream_manager::get_progress(UUID plan_id, gms::inet_address peer) const {
+    if (!_stream_bytes.count(plan_id)) {
```

Comment:
A better pattern is _stream_bytes.find(), it avoids the double-check. It doesn't matter much here but it's better practice.

Reply:
Yes, I agree. I will do it in v2 anyway.